File: Java phase-gate 1/Day2/TaskTwo.java
public class TaskTwo{
public static char checkAgeDifference(int firstScore,int secondScore,int thirdScore){
	int averageCheck = (firstScore + secondScore + thirdScore) / 3;
	if(averageCheck >= 90 && averageCheck <= 100){
		return 'A';}
	else if(averageCheck >= 80 && averageCheck < 90){
		return 'B';}
	else if(averageCheck >= 70 && averageCheck < 80){
		return 'C';}
	else if(averageCheck >= 60 && averageCheck < 70){
		return 'D';}
	else
		return 'F';
		}
		}

File: javascript phase-gate 1/Day2/TaskTwo.js
function average_score(first_score,second_score,third_score){
	average_check = (first_score + second_score + third_score) / 3
	if(average_check >= 90 && average_check <= 100){
		return 'A'}
	else if(average_check >= 80 && average_check < 90){
		return 'B'}
	else if(average_check >= 70 && average_check < 80){
		return 'C'}
	else if(average_check >= 60 && average_check < 70){
		return 'D'}
	else if(average_check >= 0 && average_check < 60){
		return 'F'}
	else {
		return 'Invalid score'}
	}
	
	console.log(average_score(65,67,65))
	
